src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt

```diff
@@ -34,6 +34,8 @@ data class SessionTemplateDto(
   val openCapacity: Int,
   @Schema(description = "day of week fpr visit", example = "MONDAY", required = false)
   val dayOfWeek: DayOfWeek?,
+  @Schema(description = "List of prison wings associated with the session template", required = false)
```

Comment:
If there are none associated this session is available to the whole prison, right? Should we mention this on the comments?

Reply:
Can I just confirm that there can be multiple wings which can be associated with the session template entry. So if Wings D & F could have visits at the same time, the PrisonWings value would hold "DF" or similar

Reply:
Yes that's correct @rj-adams

Reply:
@dhirajshettymoj Yes I think it should mention it I will add